src/status_im/contexts/communities/events.cljs

```diff
@@ -478,3 +508,28 @@
 (rf/reg-event-fx :communities/update-last-opened-at-success
  (fn [{:keys [db]} [community-id last-opened-at]]
    {:db (assoc-in db [:communities community-id :last-opened-at] last-opened-at)}))
+
+(rf/defn share-community
```

Comment:
Better use `reg-event-fx` if you're touching the code to stay consistent with the guidelines and with the rest of the green code you're introducing in this PR.

Reply:
Done